---
float32 young_modulus
